.yamllint.yml

```diff
@@ -21,3 +21,4 @@ rules:
     ignore: |
```

Comment:
Regarding the commit message:

```

Ensure testing with both initscripts and conditional NM provider

```

Proposal: Support conditional testing for NM provider

```
Current Network role can not ensure test playbook running with both
initscripts provider and conditional NM provider(the test running with NM
provider which demanding for NM mininum version or extra running
condition). This PR is intended to fix the bug.

In the mean time, enable the support for ethtool coalesce option with
initscripts provider.
```

Proposal:

```
- To properly run tests for features that are supported both in initscripts and NM but require a specific NM version or other condition, add a new test category for these tests. 
- Use the new test category to enable testing coalesce support with initscripts
```

Reply:
Commit message changed, thanks!